doc/raid.md

```diff
@@ -0,0 +1,73 @@
+# RAID setups - what works?
+
+## General considerations
+
+- How much stacking of multipath+raid+enc+lvm is allowed for /boot?
+- Avoid bcache for bootloader_partition and /boot?
+- general: / on full disk [raid] (i.e. - not a partition)?
+
+> All tests were run with tw-20180925 + libstorage-ng-4.1.37, yast2-storage-ng-4.1.19.
+
+## Terminology
+
+- raid1: raid level 1 with metadata format <= 1.0 (metadata at the end of the device)
+
+- raidX: any raid with any metadata format
+
+- bootloader_partition: partition containing the initial part of the bootloader (loaded by the system's firmware)
+    - x86, gpt: bios_grub partition
+    - x86, msdos: mbr gap
+    - efi (x86, aarch64): esp
+    - ppc64 (gpt, msdos): prep partition
+
+## Results
+
+- raid1 over full disk, partitioned, bootloader_partition on this disk
+    - non-efi: works, grub_installdevice contains N physical disks
+
+    - ! non-efi: grub2-install blocked in D-state for a very long time (3+ min)
+
+    - ! non-efi: pointing grub_installdevice to raid1 device directly results in error:
+      ```sh
+      grub2-install: error: diskfilter writes are not supported.
+      ```
+
+    - ! x86, msdos: wrong warning about missing bios_grub partition
+
+    - ! efi: (incorrect?) warning about esp on software raid
+
+    - ! efi: not working; 'shim-install --config-file=/boot/grub2/grub.cfg' fails with
+      ```sh
+      Standard output: copying /usr/lib64/efi/grub.efi to /boot/efi/EFI/opensuse/grub.efi
+      Error output: Installing for x86_64-efi platform.
+      Error output: Installation finished. No error reported.
+      Error output: invalid numeric value 
+      Error output: 
+      Status: 37
+      ```
+
+    - ! efi: no boot manager entry (in firmware) created
+
+- raidX over partitions, /boot on raidX, bootloader_partition not on raid
+    - works, grub_installdevice contains N physical disks
+    - efi: note there can be only one esp
+    - ! non-efi: need bootloader_partition on every disk constituting raidX or
+      change yast-bootloader to only install on those that have it (there should
+      be at least 1)
```

Comment:
idea was that you need it on all disk to be able to boot from whatever you want. But we also discussed that we can relax installation failure if we can install it to at least some of the disks.